package.json

```diff
@@ -7,13 +7,17 @@
     "npm": "6.9.0"
   },
   "scripts": {
-    "start": "node src/main/Main.js"
+    "start": "node src/main/Main.js",
+    "test": "mocha"
   },
   "dependencies": {
     "blessed": "^0.1.81",
     "lodash": "^4.17.11",
     "node-binance-api": "^0.9.0",
     "pino": "^5.11.1",
     "pino-pretty": "^2.5.0"
+  },
+  "devDependencies": {
+    "chai": "^4.2.0"
   }
```

Comment:
What about mocha? Do you have it installed globally on your computer and for got to add it?